WordPress/ruleset.xml

```diff
@@ -6,4 +6,10 @@
 	<rule ref="WordPress-Docs"/>
 	<rule ref="WordPress-Extra"/>
 	<rule ref="WordPress-VIP"/>
+
+	<rule ref="WordPress.PHP.DiscouragedPHPFunctions">
```

Comment:
I would add a comment here explaining why we are adding an exception or referencing to that they have been excluded from their respective rulesets.